### ISSUE TITLE
Create project card for Equity Language project

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/website#301]
### Overview
Provide collateral for the HackforLA website

### Action Items
- [ ]   Gather items
  - [x]  600 x 400 image
  - [x]  Alt image text
  - [x]  Name of project
  - [x]  A blurb about your project
  - [x]  Links (website, github, etc.)
  - [x]  What you are looking for skills wise
  - [x]  Partner(s)
  - [x]  Location
  - [x] Leadership
  - [ ] Tools
- [x]   Add project Card to website

### Resources/Instructions
See project cards on the hackforla.org website for examples

**Comment by charlottesauce** [comment]
![Gender Equity Hero](https://user-images.githubusercontent.com/45779646/75852984-03ea6700-5da2-11ea-9e5d-211bc85fae10.png)

**Comment by charlottesauce** [comment]
I totally dropped the ball.  Thanks for your patience.

**Comment by ExperimentsInHonesty** [comment]
@joelparkerhenderson shall we make the link public now?

**Comment by ExperimentsInHonesty** [comment]
@charlottesauce @joelparkerhenderson I realized that we don't list what tools you use on this project.  Can you provide?  Also, I am fine with making the [live link](https://raw.githack.com/joelparkerhenderson/site_searcher/master/index.html) appear on the homepage if you are.

**Comment by ExperimentsInHonesty** [comment]
Wrote to them in slack again.

**Comment by joelparkerhenderson** [comment]
Thanks! Tools (and practices) that will help us are JavaScript coding, Svelte framework (akin to React and Vue), AWS hosting with EC2/RDS/SES.

In parallel, my preference is aim toward self-service for these, so a project team can optionally update this information; this is because the tools that we are using may need to shift soon -- we just got a bunch of feedback that we need subject-matter creator feedback that involves Twitter, so we'll likely need some kind of media tools such as TweetDeck etc.

**Comment by ExperimentsInHonesty** [comment]
Talked to Joel about how to use the tools field.  Closing issue now.  Will revisit any changes on a new issue with new project name: writeforeveryone
